src/api/README.md

```diff
@@ -1642,6 +1642,126 @@ Result:
 }
 ```
 
+## Transaction Note APIs
+
+### Get all existing notes
+
+API sets: `READ`
+
+```
+URI: /api/v2/notes
+Method: GET
+```
+
+Gets all the existing notes to all of the transactions.
+
+Example:
+
+```sh
+curl http://127.0.0.1:6420/api/v2/notes
+```
+
+Result:
+
+```json
+{
+    "data": {
+        "a5cf149da9cab9fdff681cec9fe83983aada218a46e26292a2c977ceff5bb1a5": "note1",
+        "db6fec68266296fcf6bf98a26cf25d86c83bfc31b8248575724977d90426addd": "note2",
+        "fef07801a566c3eafd680c9d29ccc18657c600e8b9d8f2c0eb89e3c98f5019c4": "note3"
+    }
+}
+```
+
+### Get transaction note by transaction id
+
+API sets: `READ`
```

Comment:
Should be added to `NOTE` as well . Otherwise explicitly enabling/disabling `NOTE` will note affect `GET` endpoints . For instance , the way it works now disabling everything and enabling `NOTE` API set , will only enable `POST` and `DELETE` , but would be impossible to `GET` . If that's expected behavior , ok.

API sets are classifiers . Multiple values possible for a single endpoint .

Reply:
You're right, thanks. Fixed.